src/app/category/new/page.tsx

```diff
@@ -2,7 +2,7 @@
 
 import React from "react";
 import MyPageTitle from "@/components/molecules/title/my-page-title";
-import MyFormCategory from "@/components/molecules/my-form-category/my-form-category";
+import MyFormCategory from "@/components/molecules/myFormCategory/myFormCategory";
```

Comment:
same here, did you just rename it ? Why ?

Reply:
just rename